README.md

```diff
@@ -1,89 +1,98 @@
-
-Before running the entry point script it is important to set environmental variables as prerequisite.
+### Required env variables
+The following environment variables need to be defined for the scripts to run correctly (see example data under paths provided): 
 ```
-## prerequisites - export environmental variables
-## Note : These paths are used for testing purpose. While in production, we need these output files to be dumped in traditional $ATLAS_PROD/bioentity_properties/annotations 
 export ENSEMBL_JSON_PATH=/hps/nobackup2/production/ensembl/ensprod/search_dumps/release-101b/vertebrates/json (pilot file path provided by Mark from Ensembl for testing)
 export ANNOTATIONS_PATH=/ebi/microarray/home/suhaib/json_Ensembl/annotations
+export GENE_ATTRIBUTES_PATH=/ebi/microarray/home/suhaib/json_Ensembl/ensembl
 export LOG_PATH=/ebi/microarray/home/suhaib/json_Ensembl/logs
+
 ```
-The truncated test output TSV file for human is in `test/homo_sapiens.ensgene.tsv`.  
-For all other species that are defined in `./annsrcs/ensembl`, annotations can be found in this directory:
+
+### Testing the scripts
+run `atlas_annotations_ensembl_run_tests.sh` in order to execute the tests. Test outputs will be stored in `test_data/outputs`. 
+
+### Extract annotations from JSON dumps to TSV formatted files 
 
 ```
-/ebi/microarray/home/suhaib/json_Ensembl/annotations
+annotations_from_ensemb.sh <config_file>
 ```
+`config_file` is defined on a per-species basis and encodes necessary jq quries to extract corresponding fields from JSON. See `data/annsrcs/ensembl` for a full list of config files. Output file will be stored under `$ANNOTATIONS_PATH`.
 
-### Make two column TSV gene attributes file used for decoration
-
-#### Entry points
+The truncated test output TSV file for human is in `example_outputs/homo_sapiens.ensgene.tsv`.  
 
-`test/run_merge_gene_attributes.sh`
-This script is entry point to make column gene attributes files for several species. Effectively, this script trigger several LSF jobs for each species defined in `ANNOTATION_PATH` i.e output gene annotations path in the `Extract annotations from JSON dumps to TSV formatted files` task. Which means, the output files from `Extract annotations from JSON dumps to TSV formatted files` becomes input to `Make two column TSV gene attribute files used decoration`
+For species defined in `data/annsrcs/ensembl`, derived annotations can be found in this directory:
 
-Before running the entry point script it is important to set environmental variables as prerequisite.
 ```
-## prerequisites - export environmental variables
-## Note : These paths are used for testing purpose. While in production we need these output files to be dumped in traditional $ATLAS_PROD/bioentity_properties/ensembl 
-export ANNOTATIONS_PATH=/ebi/microarray/home/suhaib/json_Ensembl/annotations
-export GENE_ATTRIBUTES_PATH=/ebi/microarray/home/suhaib/json_Ensembl/ensembl
+/ebi/microarray/home/suhaib/json_Ensembl/annotations
```

Comment:
Internal path.

Reply:
I suggest that you write an internal SOP on how to run this, the you can add all these paths there.